### DISCUSSION TITLE
pyinstaller raspberry pi bullseye

### DISCUSSION THREAD

**Discussion by just9876** [pyinstaller/pyinstaller#6699] [Help]
Attempting to use pyinstaller with two python files on Bullseye operating system (python 3.9.2).

If importing from a second py module I get an exception when using an imported function.

This behaviour is not occurring on an earlier version of the Raspberry Operator System (python 3.7)

Command usage is 
pyinstaller --onefile main.py 

Any ideas

**Comment by rokm** [comment]
What error are you getting? (Please provide full backtrace).

**Reply by just9876** [reply]
Two FIles

main.py
======

import test

print("Hi")
test.test_func()

test.py
=======
def test_func():
    print("Test Func")

Build Command
===========
pyinstaller --onefile main.py

Output
=========
Hi
Traceback (most recent call last):
  File "main.py", line 4, in <module>
    test.test_func()
AttributeError: module 'test' has no attribute 'test_func'
[4102] Failed to execute script 'main' due to unhandled exception!


On Buster same files and command output is 
=======
Hi
Test Func

So I am slightly confused

**Reply by rokm** [reply]
What if you rename `test.py` to something that does not clash with a standard python module name, e.g., `mytest.py`?

`test` is also a standard [python module](https://docs.python.org/3/library/test.html), which probably ends up getting collected instead of your local `test.py`.

**Reply by just9876** [reply]
Thanks a lot, I thought it was strange that pyinstaller failed bullseye.

Also slightly strange that it works on Buster, I wonder whether test.py was not present in python3.7

**Comment by bwoodsend** [comment]
There's nothing special about having two modules so whatever you're problem is - it's something other than what your saying.
